plasma/look-and-feel/com.github.numix/AUTHORS

```diff
@@ -0,0 +1,4 @@
+------------------------ Numix Look and Feel ------------------------
```

Comment:
Because i don't know your team urls and projects names for kde... You can fix this manual on folder and desktop.theme's